packages/component-library/stories/HorizontalBarChart.story.js

```diff
@@ -225,4 +225,68 @@ export default () =>
         );
       },
       { notes }
+    )
+    .add(
+      "Example: Stacked bar chart",
+      () => {
+        const sampleStackedData = [
```

Comment:
I'm still seeing `x` and `y` here -- and I believe that there's an issue with passing in data that has property names other than `x` or `y`, as when I tried to change it to `year` and `count` in the story I ran into NaN errors.

This would make sense, because `x` and `y` have special meanings in VictoryCharts. That's why there's uglier data mapping going on for the non-stacked components. But maybe it's possible to do with the `key` prop?